docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    command: >
+      sh -c "python manage.py makemigrations &&
+             python manage.py migrate &&
+             python manage.py wait_for_db &&
```

Comment:
The `wait_for_db` command should be executed before `makemigrations` and `migrate` to ensure the database is ready before attempting to apply migrations. Consider reordering these commands.